benchmarks/schema.gql

```diff
@@ -0,0 +1,42 @@
+type Query {
+        people_dataclass: [Person!]!
+        people: [Person!]!
+        person_dataclass: [Person!]!
+        person: [Person!]!
+        users_dataclass: [User!]
+        users: [User!]
+        user_dataclass: [User!]
+        user: [User!]
+    }
+
+    type Person {
```

Comment:
Let's drop this type and query for  scalar `User` fields in "simple" test cases.

Reply:
Removed, now we only have User